google-cloud-bigtable-deps-bom/README.md

```diff
@@ -0,0 +1,44 @@
+# Dependency BOM for Google Cloud Bigtable Client
+
+This module contains a BOM that lists all of Cloud Bigtable's dependency versions. Please note that
+this BOM is complementary to google-cloud-bigtable-bom. This BOM only contains transitive dependencies,
+while google-cloud-bigtable-bom contains versions for direct google-cloud-bigtable artifacts.
+
+This BOM is primarily intended to be used by java-bigtable-hbase to keep the dependencies in sync.
+
+Example usage:
+
+[//]: # ({x-version-update-start:google-cloud-bigtable:released})
+```xml
+<dependencyManagement>
+  <dependencies>
+    <!-- Lock google-cloud-bigtable direct dependency versions -->
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-bigtable-bom</artifactId>
+      <version>1.8.0</version>
+      <type>pom</type>
+      <scope>import</scope>
+    </dependency>
+    <!-- Lock google-cloud-bigtable transitive dependency versions -->
+    <dependency>
```

Comment:
Let me know your thoughts - what if we combined these two into one (similar to how the libraries-bom contains different BOMs), or do you think that's too much?

Reply:
I want both google-cloud-bigtable and java-bigtable-hbase to use the same bom. So it can’t contain a reference for google-cloud-bigtable, otherwise google-cloud-bigtable have itself as a dep

Reply:
Gotcha, thanks for the explanation!